src/main/java/it/gov/pagopa/swclient/mil/paymentnotice/dao/PNEntity.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.swclient.mil.paymentnotice.dao;
+
+import org.bson.codecs.pojo.annotations.BsonId;
+
+import io.quarkus.mongodb.panache.common.MongoEntity;
+import it.gov.pagopa.swclient.mil.paymentnotice.bean.PspInfo;
+
+@MongoEntity(database = "mil", collection = "paymentnotice")

```

Comment:
Please, use meaningful name: this is a PSP configuration!
Rename the entity to PspConfEntity and the Mongo collection to pspconf.